app/services/bgs/payment_service.rb

```diff
@@ -21,7 +20,7 @@ def payment_history(person)
     private
 
     def empty_response
-      { payments: [], return_payments: [] }
```

Comment:
Was `return_payments` not used here?

Reply:
Turns out the return payments are actually `payments` with special attributes. We check for truthiness in the serializer and build them out.

https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/serializers/vet_payment_history_serializer.rb#L35